src/api/production.yml

```diff
@@ -18,15 +18,22 @@ services:
       - '--api.insecure=false'
       - '--providers.docker=true'
       - '--providers.docker.exposedbydefault=false'
+      - '--providers.file.directory=/config'
+      - '--providers.file.watch=true'
       - '--entrypoints.web.address=:80'
```

Comment:
Should we disable 80 in staging/prod and force people to use HTTPS always?  A lot of sources online talk about this for services vs. HTML endpoints for users.

Reply:
I think it's a good idea. I'll disable the entry point for port 80.